src/dcqc/tests/base_test.py

```diff
@@ -70,64 +61,6 @@ def get_status(self, compute_ok: bool = True) -> TestStatus:
             self._status = self.compute_status()
         return self._status
 
-    def get_files(self, staged: bool = True) -> list[File]:
-        """Get and stage files for target.
-
-        Args:
-            staged: Whether to make sure that the files are staged.
-            Defaults to True.
-
-        Returns:
-            Staged target files.
-        """
-        files = []
-        for file in self.target.files:
-            if staged:
-                file.stage()
-            files.append(file)
-        return files
-
-    def get_file(self, staged: bool = True) -> File:
```

Comment:
My rationale for this deletion is that it's simpler to have a `SingleTarget` class with `file` property than one target class with a variable number of files that raises an error when you call `get_file()` when there are more than one file. Subclasses of `BaseTarget` will add convenience functions. So, they'll present extended interfaces rather than different interfaces.